examples/tts/conf/rad-tts_dec.yaml

```diff
@@ -43,16 +44,30 @@ model:
   pitch_std: ${pitch_std}
   
   text_normalizer:
-      _target_: nemo_text_processing.text_normalization.normalize.Normalizer
-      lang: en
-      input_case: cased
-      whitelist: ${whitelist_path}
+    _target_: nemo_text_processing.text_normalization.normalize.Normalizer
+    lang: en
+    input_case: cased
+    whitelist: ${whitelist_path}
 
   text_normalizer_call_kwargs:
-      verbose: false
-      punct_pre_process: true
-      punct_post_process: true
-      
+    verbose: false
+    punct_pre_process: true
+    punct_post_process: true
+
+  text_tokenizer:
+    _target_: nemo.collections.tts.torch.tts_tokenizers.EnglishPhonemesTokenizer
+    punct: true
+    stresses: true
+    chars: true
+    apostrophe: true
+    pad_with_space: true
+    g2p:
+      _target_: nemo.collections.tts.torch.g2ps.EnglishG2p
+      phoneme_dict: ${phoneme_dict_path}
+      heteronyms: ${heteronyms_path}
+      phoneme_probability: 0.5
+      mapping_file: ${mapping_file_path}
```

Comment:
What is this mapping_file?

Reply:
This is extension for Riva - mapping file to initialize their tokenizer. Essentially, dump of 'labels' dict. This is useful because 1) for radtts ADLR->Nemo conversion case, it's the way to pass mapping to Riva without writing  unneeded tokenizer conversion; 2) For general Nemo2Riva case, it's preferred to have mapping file already prepared for export - rather than generate it from Nemo model at 'riva-build' time as that requires hooks and calling into Nemo model internals.